packages/global/components/document.marko

```diff
@@ -39,7 +39,7 @@ $ const { nativeX, site } = out.global;
   <@above-container>
     <marko-web-reveal-ad-listener />
     <marko-web-gtm-track-load-more />
-    <global-contextual-header />
+    <global-contextual-header page-type=input.container.page.for />
```

Comment:
You should import get at the top of the file `import { get } from "@parameter1/base-cms-object-path";`  & update this to `page-type=get(input, 'container.page.for')`. This will prevent it from throwing an error if container or page is undefined and return null.